cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel_avx512.hpp

```diff
@@ -520,12 +219,14 @@ vertex_similarity_result call_jaccard_default_kernel_avx512(
         else {
 #endif
             for (j = column_begin; j < diagonal; j++) {
-                const auto j_neighbor_size = g_degrees[j];
-                const auto j_neigbhors = g_vertex_neighbors + g_edge_offsets[j];
+                const auto j_neighbor_size = degrees[j];
+                const auto j_neigbhors = cols + rows_vertex[j];
                 if (!(i_neigbhors[0] > j_neigbhors[j_neighbor_size - 1]) &&
                     !(j_neigbhors[0] > i_neigbhors[i_neighbor_size - 1])) {
-                    auto intersection_value =
-                        intersection(i_neigbhors, j_neigbhors, i_neighbor_size, j_neighbor_size);
+                    auto intersection_value = preview::backend::intersection<Cpu>(i_neigbhors,
```

Comment:
const?

Reply:
Suggest applying SDL and other algorithm's required changes in another PR.